src/main/java/seedu/address/logic/messages/WeightCommandMessages.java

```diff
@@ -11,11 +9,11 @@ public class WeightCommandMessages extends Messages {
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Edits the weight (in kilograms) of the person identified "
```

Comment:
I think the weight command doesn't edit weights anymore, it can only add/remove weights now.